src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +28,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<Object> machines, Machine value);
```

Comment:
The 'fill' method should use the generic type 'T' for the 'machines' list instead of 'Object'. This ensures type safety.